manual/program/reviewer_guidelines.rst

```diff
@@ -0,0 +1,167 @@
+
+Reviewers may be asked to review submissions for talks and tutorials. Tutorials should be approachable to a broad audience in the SciPy community, so all members of the reviewer pool are eligible to review them. Please refer to the tips for tutorial review for what we'd like to see in tutorial reviews. If you feel you are not qualified to review a tutorial that you are assigned, please contact the tutorials chairs.
+
+Instructions
+------------
+
+.. image:: pretalx.jpeg
+  :width: 400
+  :alt: One does not simply Pretalx
+
+Go to https://cfp.scipy.org/orga/event/2024/reviews/ to see your assigned proposals.
+Each proposal will ask you to provide the following:
+
+- **"Overall Evaluation"** (score +2 to -2)
+-  **"Is the proposal interesting to a broad range of people in the SciPy community?"** (score +2 to -2)
+- **answer questions** (+2 to -2) on the proposal clarity and completeness
+- Finally, you will have an open response **"What did you think?"** question where we'd like you to detail your review of the submission and explain your ratings to the previous questions. 
+
+Be on the lookout for unconscious bias when reviewing proposals, and make sure to mitigate them where you can. Take some time to educate yourself about bias in the review process, and equip yourself with the tools to interrupt this bias. For more details, see the `Bias in the review process`_ section below.
+
+Please be clear and detailed in your review on why you'd like to see this proposal at the conference, or why you think it would not be a good fit; it will help track chairs in the selection process. For more details, see the `Tips for Reviewers`_ section below.
+
+Please double check that the submissions assigned to you don't present a conflict-of-interest. If you do find that you have a conflict with one of the submissions you've been assigned, please notify the track chairs in the track to which that submission belongs and they will assign a different reviewer. For more details, read the `Conflict of interest`_ section below.
+
+We expect all reviews to be conducted in accordance with the NumFOCUS Code of Conduct: https://numfocus.org/code-of-conduct. Please keep this in mind as you review submissions.
+
+
+Bias in the review process
+--------------------------
+
+.. image:: bias.jpeg
+  :width: 400
+  :alt: your bias is showing. just a bit
+
+Research shows that peer review can suffer from implicit bias—just like the hiring processes—that disadvantages women, minorities and people from less-prestigious institutions. There is also research attesting to higher quality in signed reviews compared to anonymous reviews [1],[2],[3]. For this reason, we want to educate our reviewers about bias in the review process, and equip them with tools to interrupt this bias.
```

Comment:
Could we hyperlink 

> [1],[2],[3]

(and all the otheres) to the references they correspond to below? The first thing I did when I saw them was to go click on `[1]` so having them linked would be nice. Though if this would be better suited for a follow up PR I'm happy to make this an Issue from Comment and then follow up after this is merged. :+1:

Reply:
Okay yeah, so while doable:

```rst
Research shows that peer review can suffer from implicit bias—just like the hiring processes—that disadvantages women, minorities and people from less-prestigious institutions. There is also research attesting to higher quality in signed reviews compared to anonymous reviews |ref_1|_, |ref_2|_, |ref_3|_

...

Human beings are consistently, routinely, and profoundly biased. We not only are profoundly biased, but we also almost never know we are being biased. (Interested? |ref_4|_, |ref_5|_)

...

Reviewers have multiple demands on their time, and are frequently under time-constraints to finish reviews, be aware that cognitive biases are efficient, time pressure promotes their influence on decision-making. |ref_4|_

...

First, by realizing and accepting that we all have bias, we can learn to watch for it in ourselves and help others who work with us to do the same. This process of building awareness is analogous to what happens when we step on the clutch in a manual-transmission automobile. The motor doesn't stop running (bias doesn't stop), but the car is no longer moving forward. When we are on the lookout for our biases, they are less likely to blindly dictate our decisions. |ref_6|_


.. |ref_1| replace:: [1]
.. _`ref_1`: https://github.com/scikit-hep/pyhf/releases/tag/v0.7.6

.. |ref_2| replace:: [2]
.. _`ref_2`: https://bjp.rcpsych.org/content/176/1/47/full-text.pdf+html

.. |ref_3| replace:: [3]
.. _`ref_3`: https://www.ncbi.nlm.nih.gov/pubmed/28580134/

.. |ref_4| replace:: [4]
.. _`ref_4`: https://www.amazon.com/Everyday-Bias-Identifying-Navigating-Unconscious/dp/1442230835

.. |ref_5| replace:: [5]
.. _`ref_5`: https://implicit.harvard.edu/implicit/takeatest.html

.. |ref_6| replace:: [6]
.. _`ref_6`: https://hbr.org/2015/04/3-ways-to-make-less-biased-decisions
```

![image](https://github.com/scipy-conference/scipy-conference/assets/5142394/86ef8ffd-852f-49aa-858b-504240add215)

That would be a super confusing and ugly PR suggested change, so I'll just open up a follow up PR after this is merged. :+1:

Reply:
Thanks, no worries I can add it